src/manager/k8s.service.ts

```diff
@@ -33,8 +33,6 @@ export class K8sService implements IAgentManager {
     }
 
     private async createPod(config: AgentConfig): Promise<any> {
-        const inboundTransportSplit = config.inboundTransport.split(' ');
```

Comment:
there are some other process.env....statements in the k8s.service.ts that I did not change.  I felt these were specific to k8s so it was ok to leave them here.

Reply:
Yep that makes sense